### PR TITLE
IVefiopsk8 RNBefns8

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Renaming of the record, small corrections.

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
please check.

**Comment by DenisNosnitsin1970** [comment]
please check.

File: StPetersburg/RNBefns8.xml [hunk 1]
```diff
@@ -102,47 +103,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                     <list>
                         <item xml:id="a1">
                            <desc type="Unclear">The note has been completely erased.</desc>
                            <locus from="1r" to="2v"/>
                         </item>
-                
+
                         <item xml:id="a2">
-                           <desc type="MagicFormula">The magic formular is against the headache.   <bibl>
+                           <desc type="MagicFormula">A magic formula against the headache.   <bibl>
```
Comment (thea-m) [inline StPetersburg/RNBefns8.xml]: against headache without the?

File: StPetersburg/RNBefns8.xml [hunk 2]
```diff
@@ -152,65 +153,70 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
                <history>
                   <origin>
                      <origDate notAfter="1859"/>
-                     <desc> The manuscript was aquired by <persName ref="PRS11337Tischendorf"/>  possibly during his third trip to the Orient, 
-                in <date>1859</date>.</desc>
                      <note>
                         <persName ref="PRS9602Turaev"/> did not propose a dating for the manuscript </note>
                   </origin>
+                  <provenance>The manuscript was aquired by <persName ref="PRS11337Tischendorf"/>
```
Comment (thea-m) [inline StPetersburg/RNBefns8.xml]: acquired

File: StPetersburg/RNBefns8.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply StPetersburg/RNBefns8.xml]: Ok.

File: StPetersburg/RNBefns8.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply StPetersburg/RNBefns8.xml]: Ok.

**Comment by DenisNosnitsin1970** [comment]
Attended.

**Pull request merged**